tests/integration/docidcache/test_crud_cache.py

```diff
@@ -0,0 +1,257 @@
+import os
+
+import numpy as np
+import pytest
+
+from jina import Flow, Document
+from jina.executors.indexers import BaseIndexer
+from jina.executors.indexers.cache import DocIDCache
+from jina.executors.indexers.keyvalue import BinaryPbIndexer
+from jina.executors.indexers.vector import NumpyIndexer
+from tests.integration.incremental_indexing import random_workspace
```

Comment:
not a fan of adding dependencies between tests, better duplicate code

Reply:
Removed. Didn't notice it was defined there when I did auto-import